pkg/apis/api/types.go

```diff
@@ -1102,9 +1116,22 @@ type ComponentReportTriageIncidentSummary struct {
 	TriagedIncidents []TriagedIncident `json:"incidents"`
 }
 
+// TestRegression is used for rows in the test_regressions table and is used to track when we detect test
+// regressions opening and closing.
+type TestRegression struct {
+	Release      string                 `bigquery:"release" json:"release"`
+	TestID       string                 `bigquery:"test_id" json:"test_id"`
+	TestName     bigquery.NullString    `bigquery:"test_name" json:"test_name"`
+	RegressionID string                 `bigquery:"regression_id" json:"regression_id"`
+	Opened       time.Time              `bigquery:"opened" json:"opened"`
+	Closed       bigquery.NullTimestamp `bigquery:"closed" json:"closed"`
+	Variants     []TriagedVariant       `bigquery:"variants" json:"variants"`
```

Comment:
Nit, since this has become more than just Triage, I'd probably rename this to Variant

Reply:
Sadly already claimed by sippy classic, but I made it ComponentReportVariant.